prow/cmd/branchprotector/protect.go

```diff
@@ -136,12 +136,11 @@ func main() {
 	cfg := ca.Config()
 	cfg.BranchProtectionWarnings(logrus.NewEntry(logrus.StandardLogger()), cfg.PresubmitsStatic)
 
-	secretAgent := &secret.Agent{}
-	if err := secretAgent.Start(nil); err != nil {
```

Comment:
Calls like this were just an initialization and have no effect with the singleton since it's initialized at program start